### ISSUE TITLE
Errors when enabling Jetpack

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#1772]
- Multisite network
- New site in that network
- Running Jetpack master as of Saturday, March 14, 11:53 UTC

```
Warning: array_keys() expects parameter 1 to be array, string given in /wp-content/plugins/jetpack/class.jetpack-options.php on line 157

Warning: array_diff(): Argument #1 is not an array in /wp-content/plugins/jetpack/class.jetpack-options.php on line 159

Warning: Invalid argument supplied for foreach() in /wp-content/plugins/jetpack/class.jetpack-options.php on line 159

Warning: Invalid argument supplied for foreach() in /wp-content/plugins/jetpack/class.jetpack-options.php on line 164

Warning: Cannot modify header information - headers already sent by (output started at /wp-content/plugins/jetpack/class.jetpack-options.php:157) in /wp-includes/option.php on line 772

Warning: Cannot modify header information - headers already sent by (output started at /wp-content/plugins/jetpack/class.jetpack-options.php:157) in /wp-includes/option.php on line 773
```

cc @dereksmart Anything from 5810dcc41261de8afb2cfb57e13b2d7936df9b1e that may have caused that problem?

**Comment by dereksmart** [comment]
cc @lezama because It looks like he originally authored the changes to that file.  Sorry about that.  Good reason to rebase instead of merge here.

**Comment by enejb** [comment]
Thanks for reporting this @jeherve! Should be fixed in #1773
